### DISCUSSION TITLE
How do you use CO-OP Checkpoints?

### DISCUSSION THREAD

**Discussion by Flammable-Nate** [BEEmod/BEE2-items#4066] [Tech Support / Q&A] [answered]
I've been doing a coop map using beemod and I have no idea how to use checkpoints because no matter where I use it it's in a "Invalid Spot"

**Comment by slambo311** [comment]
If it shows a red outline, make sure that the tiles outlined are filled in, along with the tiles around them.
Also, what version of beemod are you using?

**Comment by vrad-exe** [comment, marked as answer]
The blocks surrounding the exit doors on the opposite side of the checkpoint also need to be open.
